boards/xtensa/intel_adsp_cavs18/doc/generic_intel_adsp.rst

```diff
@@ -159,3 +170,68 @@ field. For example:
        - --rimage-tool=/home/user/sof/rimage/rimage
        - --config-dir=/home/user/sof/rimage/config/
        - --key=/home/user/sof/keys/otc_private_key_3k.pem
+
+
+
+Get Service by Port
+*******************
+
+In some case, we might need to access the service of the CAVS server by
+specifying a port. There are two service working in the server:
```

Comment:
Please make it clearer that the default values will work just fine for most people and that they can skip reading this very long section and extra overhead.

Reply:
Yes, updated it.